dev/com.ibm.ws.security.kerberos.java8/src/com/ibm/ws/security/kerberos/internal/KerberosExtServiceImpl.java

```diff
@@ -145,4 +138,26 @@ public boolean isS4U2selfEnable() {
     public boolean isS4U2proxyEnable() {
         return (kerberosExtConfig != null ? kerberosExtConfig.isS4U2proxyEnable() : false);
     }
+
+    /**
+     * Register the webcontainer's default delegation provider.
+     *
+     * @param cc
+     */
+    private void registerKrb5HelperJdk(ComponentContext cc) {
+        Dictionary<String, Object> props = new Hashtable<String, Object>();
+        if (Krb5LoginModuleWrapper.isIBMLoginModuleAvailable()) {
```

Comment:
Krb5LoginModuleWrapper statically caches the result of this method, would it be better to look at the cached result rather than calling this method each time?

Reply:
See my suggestion here: https://github.com/OpenLiberty/open-liberty/issues/18054